ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -35,9 +35,9 @@ static int HandleFileOpOperation(
 static int GetPid(vfs_context_t context);
 
 static uint32_t ReadVNodeFileFlags(vnode_t vn, vfs_context_t context);
-static inline bool FileFlagsBitIsSet(uint32_t fileFlags, uint32_t bit);
-static inline bool FileIsFlaggedAsInRoot(vnode_t vnode, vfs_context_t context);
-static inline bool ActionBitIsSet(kauth_action_t action, kauth_action_t mask);
```

Comment:
Just curious, why did you take the `inline`s back out?

Reply:
We can add them back.  

I remembered while working on the code that the `inline` is only required on the definition and not the declaration (see: https://isocpp.org/wiki/faq/inline-functions#inline-nonmember-fns).

But it doesn't hurt anything to leave them (and makes it clearer that they are expected to be inline) so I can add them back in.